pkg/minikube/cluster/commands.go

```diff
@@ -0,0 +1,34 @@
+package cluster
+
+var startCommand = `
+sudo killall localkube || true
+# Download and install localkube, if it doesn't exist yet.
+if [ ! -e /usr/local/bin/localkube ]; then
+	sudo curl -L %s -o /usr/local/bin/localkube
+	sudo chmod a+x /usr/local/bin/localkube;
+fi
+# Fetch easy-rsa.
+sudo mkdir -p /srv/kubernetes/certs && sudo chmod -R 777 /srv
+if [ ! -e easy-rsa.tar.gz ]; then
```

Comment:
I tried a few different ways, using https://github.com/SvenDowideit/generate_cert/blob/master/generate_cert.go but was unable to get anything working in pure go :(